scss/_mixins.scss

```diff
@@ -0,0 +1,269 @@
+@mixin oTopperThemeElements {
+	.o-topper--has-headshot {
+		@include oTopperHasHeadshot;
+	}
+
+	.o-topper__opinion-genre {
+		@include oTopperOpinionGenre;
+	}
+
+	.o-topper__headshot {
+		@include oTopperHeadshot;
+	}
+
+	.o-topper__headshot-image {
+		@include oTopperHeadshotImage;
+	}
+}
+
+@mixin oTopperThemes() {
+	.o-topper--branded {
+		@include oTopperThemeBranded;
+	}
+
+	.o-topper--opinion {
+		@include oTopperThemeOpinion;
+	}
+
+	.o-topper--full-bleed-offset {
+		@include oTopperThemeFullBleedOffset;
+	}
+
+	.o-topper--split-text-left,
+	.o-topper--split-text-center {
+		@include oTopperThemeSplitText;
+	}
+
+	.o-topper--full-bleed-image-center,
+	.o-topper--full-bleed-image-left {
+		@include oTopperThemeFullBleedImage;
+	}
+
+	.o-topper--package {
+		@include oTopperThemePackage;
+	}
+
+	.o-topper--package-extra,
+	.o-topper--package-extra-wide {
+		@include oTopperThemePackageExtra;
+	}
+
+	.o-topper--package-special-report {
+		@include oTopperThemePackageSpecialReport;
+	}
+
+	.o-topper--news-package {
+		@include oTopperThemeNewsPackage;
+	}
+
+	@include oTopperThemeElements;
```

Comment:
Theme elements which are shared may be modified be a theme so this should be output before the themes themselves?

Reply:
yep, better for it to go first, although none of the elements actually are modified by themes (yet).

maybe the mixin is misnamed or needs a clarifying comment, i'm not sure it's clear these are "elements which only exist in a themed topper"